woocommerce/woocommerce-android
# Second Push notification won't open correct screen

### Description


Context: peaMlT-7K-p2

Clicking on Order notification should always open a detail of the order. However, it seems that this flow doesn’t work in some cases.

1. Open Product Detail
2. Put the app into background
3. Order something on the web and wait for the notification
4. Tap on the notification
5. Notice order detail is shown
6. Navigate to a product detail
7. Order something on the web and wait for the notification
8. Tap on the notification
9. Notice Product detail is displayed instead of the order detail



### Testing instructions


Follow the steps above on trunk and make sure you can reproduce the issue (to me it's reproducible 100% of the times)
Then, on this branch, follow the steps above and make sure the second notification opens the Order details for the notification and not the previous open product details screen.

### Images/gif
